dll/win32/shell32/shelldesktop/CDesktopBrowser.cpp

```diff
@@ -390,6 +390,20 @@ LRESULT CDesktopBrowser::OnSize(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &b
 
 LRESULT CDesktopBrowser::OnSettingChange(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandled)
 {
+    // RegenerateUserEnvironment
+    typedef BOOL (WINAPI *REGENERATEUSERENVIRONMENT)(LPVOID *, BOOL);
+
+    HMODULE hShell32 = GetModuleHandleA("shell32");
+
+    REGENERATEUSERENVIRONMENT pRegenerateUserEnvironment;
+    pRegenerateUserEnvironment = (REGENERATEUSERENVIRONMENT)GetProcAddress(hShell32, "RegenerateUserEnvironment");
```

Comment:
No need to load the API, since you are using it from within shell32.

Reply:
base/shell/explorer/CMakeFiles/explorer.dir/traywnd.cpp.obj:traywnd.cpp:(.text$_ZN11CTrayWindow20ProcessWindowMessageEP6HWND__jjlRlm[__ZN11CTrayWindow20ProcessWindowMessageEP6HWND__jjlRlm]+0x2f42): undefined reference to `RegenerateUserEnvironment(void**, int)@8'

Reply:
Directly called RegenerateUserEnvironment.